modules/ROOT/pages/modularizing-your-configuration-files-for-team-development.adoc

```diff
@@ -4,51 +4,44 @@ include::_attributes.adoc[]
+* Side-by-side: Provide a list of independent configuration files to load.
+* Imported: Have one configuration file import several others, which in-turn, can
 import other files.
 
 In practice, it is common to use both approaches simultaneously.
 
-Don't forget that all the configuration files end up loaded in the same context;
-therefore you should be careful and use unique names for all your configuration
-elements. Mule will refuse to load an application whose configuration files contain
- name conflicts.
-
-How can you determine what constitutes good separation lines between configuration
- fragments? Here are a few rules of thumb:
-
-* Business domains usually form a natural border that can be used to separated
-configuration elements +
- * Keeping together elements that have similar reasons for change reduces the risk
-  of impacting unrelated aspects of your application +
- * Technical aspects, like administrative components, security or external system
- configurations, define good lines of demarcation +
- * Extracting a side-by-side connector configuration facilitates functional testing.
- Note that it is not intended to take care of environment specific connector
- configuration, which is dealt with properties files +
- * And, last but not least, re-use across teams and projects
+Because configuration files end up loaded in the same context,
+you must use unique names for the configuration elements. Mule will not load an application if its configuration files have name conflicts.
+
+Here are some suggested ways to determine where to split configuration elements into separate files and which elements to keep together:
+
+* Business domains usually form a natural border that you can use to separate
+configuration elements.
+* Keep together elements that have similar reasons for change. This reduces the risk of impacting unrelated aspects of your application.
+* Technical elements, such as administrative components, security, and external system configurations determine good separation lines.
+* Separate a side-by-side connector configuration to facilitate functional testing.
+(This does not apply to environment-specific connector configurations, which use properties files.)
+* Consider re-use across teams and projects when you separate configuration elements.
 
 [NOTE]
-By default, all present XML configuration files are considered configurations
-when an application is packaged and loaded in the syde-by-side fashion.
+By default, Mule treats all present XML configuration files as configurations
+when an application is packaged and loaded in the side-by-side fashion.
 
 == Imported configuration files
 
 Mule has a special configuration component for importing configuration files into
-each other called `import`. This component take a `file` parameter which should
-contain the reference to the desired configuration file.
+each other called `import`. This component take a `file` parameter, which contains
```

Comment:
Maybe reword: "Mule has a special `import` configuration component for ..."

Reply:
"component take" -> "component takes"

Reply:
OK